docs/scenarios_authoring/Intro.md

```diff
@@ -46,36 +46,37 @@ SpEL [Spring Expression Language](https://docs.spring.io/spring-framework/docs/3
 | 42 + 2               | 44                             | Integer              |
 | 'AA' + 'BB'          | "AABB"                         | String               |
 
-SpEL is used in Nussknacker to access data processed by a node and expand node's configuration capabilities. Some examples:
+SpEL is used in Nussknacker to access data processed by a node or expand a node's configurability. For instance:
 
 
-* create boolean expression (for example in filters) based on logical or relational (equal, greater than, etc) operators
-* access, query and manipulate fields of the data record
+* create a boolean expression (for example in filters) based on logical or relational (equal, greater than, etc) operators
+* access, query and manipulate fields of a data record
 * format data records written to sinks
-* provide helper functions like date and time, access to system variables
+* provide helper functions like getting current date and time 
+* access to system variables
 * and many more.
 
-The [SpEL Cheat Sheet page](Spel.md)  provides an exhaustive list of examples of how to write expressions with SpEL.
+The [SpEL Cheat Sheet page](Spel.md)  provides an exhaustive list of examples on how to write expressions with SpEL.
 
 &nbsp;
 ## Data Types
 
-Every SpEL expression returns a value of one of the predefined SpEL data types, like integer, double or boolean, map, etc. Data types in Nussknacker can be a confusing aspect at the beginning, as depending on the context in which data are processed or displayed, different data type schemes are in use - please refer to the [SpEL Cheat Sheet page](Spel.md#data-types-and-structures) for more information. 
+Every SpEL expression returns a value of one of the predefined SpEL data types, like integer, double, boolean, map, etc. Data types in Nussknacker can be a confusing aspect at the beginning, as depending on the context in which data is processed or displayed, different data type schemes will be used - please refer to the [SpEL Cheat Sheet page](Spel.md#data-types-and-structures) for more information. 
 
 In some contexts data type conversions may be necessary - conversion functions are described [here](Spel.md#type-conversions).
 
 &nbsp;
 ## Variables
 
-Nussknacker uses variables as containers for data. Variables have to be declared; a `variable` or `record-variable` component are used for this. Once declared, a hash sign `"#"` is used to refer to a variable from a SpEL expression. Variables are attributes of the data record, they do not exist by themselves. 
+Nussknacker uses variables as containers for data. Variables have to be declared; a `variable` or `record-variable` component is used for this. Once declared, a hash sign `"#"` is used to refer to a variable from a SpEL expression. Variables are part of a data record, they do not exist by themselves. 
```

Comment:
_Variables are part of a data record_ vs original _attributes of a data record_. What is better? Which of these words carries idea of adding new variables to the data record? Btw, what is wrong with term _attribute_?

Reply:
This is just a matter of taste, I think both are technically ok.

Reply:
Updated to attributes.